2021-11-27 17:58:44,919 INFO    MainThread:4129 [wandb_setup.py:_flush():71] setting env: {}
2021-11-27 17:58:44,919 INFO    MainThread:4129 [wandb_setup.py:_flush():71] setting login settings: {}
2021-11-27 17:58:44,919 INFO    MainThread:4129 [wandb_setup.py:_flush():71] setting login settings: {}
2021-11-27 17:58:44,919 INFO    MainThread:4129 [wandb_init.py:_log_setup():371] Logging user logs to /geoinfo_vol1/home/z/h/zhao2/vit-keras/wandb/run-20211127_175844-214memw7/logs/debug.log
2021-11-27 17:58:44,920 INFO    MainThread:4129 [wandb_init.py:_log_setup():372] Logging internal logs to /geoinfo_vol1/home/z/h/zhao2/vit-keras/wandb/run-20211127_175844-214memw7/logs/debug-internal.log
2021-11-27 17:58:44,920 INFO    MainThread:4129 [wandb_init.py:init():404] calling init triggers
2021-11-27 17:58:44,920 INFO    MainThread:4129 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2021-11-27 17:58:44,920 INFO    MainThread:4129 [wandb_init.py:init():449] starting backend
2021-11-27 17:58:44,921 INFO    MainThread:4129 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-11-27 17:58:44,949 INFO    MainThread:4129 [backend.py:ensure_launched():199] starting backend process...
2021-11-27 17:58:44,970 INFO    MainThread:4129 [backend.py:ensure_launched():205] started backend process with pid: 4264
2021-11-27 17:58:44,972 INFO    MainThread:4129 [wandb_init.py:init():458] backend started and connected
2021-11-27 17:58:44,974 INFO    MainThread:4129 [wandb_init.py:init():519] updated telemetry
2021-11-27 17:58:45,028 INFO    MainThread:4129 [wandb_init.py:init():549] communicating current version
2021-11-27 17:58:46,210 INFO    MainThread:4129 [wandb_init.py:init():554] got version response 
2021-11-27 17:58:46,211 INFO    MainThread:4129 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2021-11-27 17:58:46,466 INFO    MainThread:4129 [wandb_init.py:init():592] starting run threads in backend
2021-11-27 17:58:51,477 INFO    MainThread:4129 [wandb_run.py:_console_start():1816] atexit reg
2021-11-27 17:58:51,478 INFO    MainThread:4129 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.REDIRECT
2021-11-27 17:58:51,478 INFO    MainThread:4129 [wandb_run.py:_redirect():1695] Redirecting console.
2021-11-27 17:58:51,481 INFO    MainThread:4129 [wandb_run.py:_redirect():1751] Redirects installed.
2021-11-27 17:58:51,481 INFO    MainThread:4129 [wandb_init.py:init():619] run started, returning control to user process
2021-11-27 17:59:28,942 INFO    MainThread:4129 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 1
2021-11-27 17:59:28,943 INFO    MainThread:4129 [wandb_run.py:_restore():1758] restore
2021-11-27 17:59:31,394 INFO    MainThread:4129 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 765
  total_bytes: 765
}

2021-11-27 17:59:31,622 INFO    MainThread:4129 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 765
  total_bytes: 765
}

2021-11-27 17:59:31,870 INFO    MainThread:4129 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 765
  total_bytes: 765
}

2021-11-27 17:59:32,595 INFO    MainThread:4129 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 765
  total_bytes: 163303
}

2021-11-27 17:59:32,698 INFO    MainThread:4129 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 765
  total_bytes: 165952
}

2021-11-27 17:59:32,801 INFO    MainThread:4129 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 765
  total_bytes: 165952
}

2021-11-27 17:59:32,902 INFO    MainThread:4129 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 765
  total_bytes: 165952
}

2021-11-27 17:59:33,004 INFO    MainThread:4129 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 765
  total_bytes: 165952
}

2021-11-27 17:59:33,105 INFO    MainThread:4129 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 765
  total_bytes: 165952
}

2021-11-27 17:59:33,206 INFO    MainThread:4129 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 765
  total_bytes: 165952
}

2021-11-27 17:59:33,307 INFO    MainThread:4129 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 765
  total_bytes: 165952
}

2021-11-27 17:59:33,409 INFO    MainThread:4129 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 765
  total_bytes: 165952
}

2021-11-27 17:59:33,510 INFO    MainThread:4129 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 765
  total_bytes: 165952
}

2021-11-27 17:59:33,611 INFO    MainThread:4129 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 765
  total_bytes: 165952
}

2021-11-27 17:59:33,712 INFO    MainThread:4129 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 765
  total_bytes: 165952
}

2021-11-27 17:59:33,813 INFO    MainThread:4129 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 765
  total_bytes: 165952
}

2021-11-27 17:59:33,915 INFO    MainThread:4129 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 765
  total_bytes: 165952
}

2021-11-27 17:59:34,016 INFO    MainThread:4129 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 765
  total_bytes: 165952
}

2021-11-27 17:59:34,117 INFO    MainThread:4129 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 765
  total_bytes: 165952
}

2021-11-27 17:59:34,218 INFO    MainThread:4129 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 2146
  total_bytes: 165952
}

2021-11-27 17:59:34,319 INFO    MainThread:4129 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 2146
  total_bytes: 165952
}

2021-11-27 17:59:34,421 INFO    MainThread:4129 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 165952
  total_bytes: 165952
}

2021-11-27 17:59:34,523 INFO    MainThread:4129 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 165952
  total_bytes: 165952
}

2021-11-27 17:59:34,625 INFO    MainThread:4129 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 165952
  total_bytes: 165952
}

2021-11-27 17:59:34,727 INFO    MainThread:4129 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 165952
  total_bytes: 165952
}

2021-11-27 17:59:34,830 INFO    MainThread:4129 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 165952
  total_bytes: 165952
}

2021-11-27 17:59:34,932 INFO    MainThread:4129 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 165952
  total_bytes: 165952
}

2021-11-27 17:59:35,035 INFO    MainThread:4129 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 165952
  total_bytes: 165952
}

2021-11-27 17:59:35,137 INFO    MainThread:4129 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 165952
  total_bytes: 165952
}

2021-11-27 17:59:35,375 INFO    MainThread:4129 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 165952
  total_bytes: 165952
}

2021-11-27 17:59:35,896 INFO    MainThread:4129 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 165952
  total_bytes: 165952
}
local_info {
}

2021-11-27 17:59:37,223 INFO    MainThread:4129 [wandb_run.py:_append_files():2184] logging synced files
